types/japanese-holidays/index.d.ts

```diff
@@ -3,10 +3,33 @@
 // Definitions by: syamatoo <https://github.com/syamatoo>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
+export as namespace JapaneseHolidays;
+
 export function isHoliday(date: Date, furikae?: boolean): string | undefined;
 export function isHolidayAt(date: Date, furikae?: boolean): string | undefined;
 export function getHolidaysOf(year: number, furikae?: boolean): Holiday[];
 
+export function getJDate(d: Date): number;
+export function getJDay(d: Date): number;
+export function getJFullYear(d: Date): number;
+export function getJHours(d: Date): number;
+export function getJMinutes(d: Date): number;
+export function getJMonth(d: Date): number;
+export function j2u(date: Date): Date;
+export function jDate(y: number, m: number, d: number): Date;
+export function shiftDate(
+    date: Date,
+    year: number,
+    mon: number,
+    day: number,
+    hour: number,
+    min: number,
+    sec: number,
+    msec: number,
+): Date;
+export function u2j(date: Date): Date;
```

Comment:
Same thing. This is not important but implementation's argument name is `d` instead of `date`.